--- /opt/influxdb/versions/0.9.4.2/scripts/init.sh.orig	2015-11-13 10:21:49.781343738 +0000
+++ /opt/influxdb/versions/0.9.4.2/scripts/init.sh	2015-11-13 10:27:34.114134446 +0000
@@ -34,11 +34,11 @@
 
 # Daemon name, where is the actual executable
 # If the daemon is not there, then exit.
-DAEMON=/opt/influxdb/influxd
+DAEMON=/usr/bin/influxd
 [ -x $DAEMON ] || exit 5
 
 # Configuration file
-CONFIG=/etc/opt/influxdb/influxdb.conf
+CONFIG=/etc/influxdb/influxdb.conf
 
 # PID file for the daemon
 PIDFILE=/var/run/influxdb/influxd.pid
