--- /opt/influxdb/versions/0.9.4.2/scripts/init.sh.orig	2015-11-13 10:21:49.781343738 +0000
+++ /opt/influxdb/versions/0.9.4.2/scripts/init.sh	2015-11-13 12:08:10.650277378 +0000
@@ -145,7 +145,7 @@
             fi
         # if PID file does not exist, check if writable
         else
-            su -c "touch $PIDFILE" $USER > /dev/null 2>&1
+            runuser -s /bin/sh $USER -c "touch $PIDFILE > /dev/null 2>&1"
             if [ $? -ne 0 ]; then
                 log_failure_msg "$PIDFILE not writable, check permissions"
                 exit 5
@@ -164,7 +164,7 @@
         if which start-stop-daemon > /dev/null 2>&1; then
             start-stop-daemon --chuid $GROUP:$USER --start --quiet --pidfile $PIDFILE --exec $DAEMON -- -pidfile $PIDFILE -config $CONFIG $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &
         else
-            su -s /bin/sh -c "nohup $DAEMON -pidfile $PIDFILE -config $CONFIG $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &" $USER
+            runuser -s /bin/sh $USER -c "nohup $DAEMON -pidfile $PIDFILE -config $CONFIG $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &"
         fi
         log_success_msg "$NAME process was started"
         ;;
